synapse/storage/databases/main/event_push_actions.py

```diff
@@ -294,8 +294,8 @@ def add_thread_id_txn(
             # Update the thread ID for any of those rows.
             max_stream_ordering = rows[-1][0]
 
-            sql = f"""
-            UPDATE {table_name}
+            sql = """
+            UPDATE event_push_actions
             SET thread_id = 'main'
             WHERE ? < stream_ordering AND stream_ordering <= ? AND thread_id IS NULL
```

Comment:
(`"event_push_actions_stream_ordering" btree (stream_ordering, user_id)` so this one is OK)